src/web/views/Content/Select/style.scss

```diff
@@ -0,0 +1,3 @@
+.create-item {
```

Comment:
it is idiomatic for CSS modules to use `camelCase` to simplify usage from JS side

Reply:
Thanks, I'm also using this notation everywhere in the project and find it a mess in JS.

Reply:
Makes sense, fixed that.